libcanard/canard.h

```diff
@@ -268,7 +268,7 @@ typedef struct
 /// This is an intentional time-memory trade-off: use a large look-up table to ensure predictable temporal properties.
 typedef struct CanardRxSubscription
 {
-    struct CanardRxSubscription* _next;  ///< Internal use only.
+    struct CanardRxSubscription* next;  /// Read-only DO NOT MODIFY THIS
```

Comment:
```suggestion
    struct CanardRxSubscription* next;  ///< Read-only DO NOT MODIFY THIS
```

This angle bracket is a Doxygen thing, let's keep it.